xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -229,13 +230,19 @@
         <Parameter Name="numberOfCharactersWritten" Type="System.Int32&amp;" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="unicodeScalar">To be added.</param>
-        <param name="buffer">To be added.</param>
-        <param name="bufferLength">To be added.</param>
-        <param name="numberOfCharactersWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="unicodeScalar">Unicode scalar.</param>
+        <param name="buffer">The destination of the encoded text.</param>
```

Comment:
The destination of the encoded text --> A pointer to the buffer to which to write the encoded text